cdap-ui/app/hydrator/leftpanel.less

```diff
@@ -434,11 +442,8 @@ body.theme-cdap.state-hydrator-create {
                 border: 1px solid transparent;
                 color: #666666;
                 line-height: 30px;
-                margin-bottom: 6px;
+                margin-bottom: 15px;
```

Comment:
I think this might make the popover/tooltip to not be center aligned with the icon in the left panel. I am noticing these

<img width="429" alt="screen shot 2017-03-29 at 2 25 07 pm" src="https://cloud.githubusercontent.com/assets/1452845/24477362/92cafc2c-148b-11e7-8e32-3653415e75c8.png">
<img width="721" alt="screen shot 2017-03-29 at 2 24 57 pm" src="https://cloud.githubusercontent.com/assets/1452845/24477363/92cdbd22-148b-11e7-97a0-70fcd9b0f402.png">

@leacd Any suggestions?  As to what where the popover should be shown. In the previous designs plugins in the left panel had borders and the tooltips aligned with that. Since we are no longer having borders should we position the tooltip to the icon or the text of the plugin?